internal/utils/trace_verbose.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 API7.ai, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"github.com/api7/cloud-cli/internal/output"
+	"github.com/api7/cloud-go-sdk"
+	"sync"
+	"time"
+)
+
+var WantExit bool
+var VerboseWg sync.WaitGroup
+
+func VerboseGoroutine(traceChan <-chan *cloud.TraceSeries) {
+	VerboseWg.Add(1)
+	defer VerboseWg.Done()
+
+	for {
+		select {
+		case data := <-traceChan:
+			DumpTrace(data)
+		default:
+			if WantExit {
```

Comment:
That's a tricky way. Why not use the context to control the exit of this goroutine.

Reply:
Done